app/src/main/java/com/lightricks/feedexercise/database/FeedDatabase.kt

```diff
@@ -1,5 +1,23 @@
 package com.lightricks.feedexercise.database
 
-/**
- * todo: add the abstract class that extents RoomDatabase here
- */
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+
+
+@Database(entities = arrayOf(Entity::class), version = 1)
+abstract class FeedDatabase : RoomDatabase() {
+
+    abstract fun feedDao(): FeedDao
+
+    companion object {
+        fun getDatabase(context: Context): FeedDatabase {
```

Comment:
A new DB is created each time `getDatabase()` is called. This is bad for performance and not thread-safe. Please, look up how to do it using the Singleton Pattern and synchronization.